ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -26,6 +27,44 @@ def argmax(a, axis=None, out=None, keepdims=False):
     return ivy.argmax(a, axis=axis, keepdims=keepdims, out=out, dtype=ivy.int64)
 
 
+# argmin
+@to_ivy_arrays_and_back
+@with_supported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("int16", "int32", "int64")}},
+    "paddle",
+)
+def argmin(a, axis=None, out=None, keepdims=None):
+    if a is not None:
+        if isinstance(a, list):
+            if all(isinstance(elem, ivy.Array) for elem in a):
+                if len(a) == 1:
+                    a = a[0]
+                else:
+                    return [
+                        ivy.argmin(
+                            ivy.to_native_arrays(elem),
+                            axis=axis,
+                            out=out,
+                            keepdims=keepdims,
+                        )
+                        for elem in a
+                    ]
+            else:
+                raise ValueError(
+                    "Input 'a' must be an Ivy array or a list of Ivy arrays."
+                )
+
+        if not isinstance(a, ivy.Array):
+            raise TypeError("Input 'a' must be an array.")
+
+        if a.size == 0:
+            raise ValueError("Input 'a' must not be empty.")
+
+        return ivy.argmin(a, axis=axis, out=out, keepdims=keepdims)
+    else:
+        raise ValueError("argmin takes at least 1 argument.")
+
```

Comment:
Hi @MuhammedAshraf2020 . The reason why I expanded on the function was because I got some errors when I simplified it like some others in the same file, especially the part that deals with arrays. I also think it might be beneficial to have more expanded codes that raise errors if the wrong input is passed through.